armi/materials/fakeMaterial.py

```diff
@@ -0,0 +1,37 @@
+"""fake test materials for unit tests in armi.reactor.converters.tests.test_axialExpansionChanger"""
+from armi.materials.ht9 import HT9
+from armi.utils import units
+
+
+class Fake(HT9):
+    """
+    Fake material based on HT9, used to verify
+    assembly axial expansion
+    """
+
+    name = "Fake"
+
+    def __init__(self):
+        HT9.__init__(self)
+
+    def linearExpansionPercent(self, Tk=None, Tc=None):
+        """ A fake linear expansion percent"""
+        Tc = units.getTc(Tc, Tk)
+        return 0.02 * Tc
+
+
+class FakeException(HT9):
```

Comment:
Could you explain what exceptions will be raised with using this class and where they come about? This looks pretty similar to the fake HT9 above.

Reply:
Please put that description in thr class docstrings as a note.

```
Notes
---------
Description of why this results in an exception and what it's purpose is.
```

Reply:
Addressed.

https://github.com/terrapower/armi/pull/606/commits/9794c07a10080fb7c9a08e9a450cca37413899c4